front/public/locales/en/translation.json

```diff
@@ -170,10 +170,13 @@
       "BufferStop": "Buffer stop",
       "Detector": "Detector",
       "Electrification": "Electrification",
+      "NeutralSection": "Neutral section",
+      "OperationalPoint": "Operational point",
```

Comment:
I think these are technically never used: for instance, the editor has no way to edit an operational point. (But OTOH adding 3 unused keys isn't a big deal.)